src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -184,7 +184,7 @@ public static void ServerCertificateValidation_EchoString()
             CustomBinding binding = new CustomBinding(new TextMessageEncodingBindingElement(MessageVersion.Soap11, Encoding.UTF8), new HttpsTransportBindingElement());
 
             endpointAddress = new EndpointAddress(new Uri(Endpoints.Https_DefaultBinding_Address));
-            clientCertThumb = ServiceUtilHelper.LocalCertThumbprint;
+            clientCertThumb = ServiceUtilHelper.ClientCertificate.Thumbprint;
```

Comment:
Reviewers: ServiceUtilHelper.ClientCertificate is guaranteed to be non-null if Client_Certificate_Installed is declared in [ConditionalFact].